sdk/clientcore/core/src/main/java/io/clientcore/core/observability/AttributeType.java

```diff
@@ -0,0 +1,12 @@
+package io.clientcore.core.observability;
+
+public enum AttributeType {
```

Comment:
Does this need to be public at this time? All usages are for package-private methods.

Reply:
good point, not all of these apis need to be public - I'll update them